README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
Fair call.  I wasn't aware of that directive.  If we can achieve a similar effect with existing code that is standards compliant I would certainly support that.